frontend/src/components/Organization.tsx

```diff
@@ -58,15 +58,19 @@ class Organization extends React.Component<OrgProps, OrgState> {
       return <Loader />;
     }
 
-    const org = this.state.org;
-
-    if (this.state.errors.length > 0 || org === undefined) {
+    if (this.state.errors.length > 0) {
       return <Errors errors={this.state.errors} />;
     }
 
+    if (this.state.org === undefined) {
+      return (
+        <Errors errors={["Internal error, 'org' should not be undefined"]} />
```

Comment:
are we exposing this error to the user? can he fix it?

Reply:
If we reach this point it's because we introduced a bug, this shouldn't happen. In this case, I'd say it's ok to render the error so the user can report it to us and describe the steps they made to reproduce it.
Otherwise the button will fail silently, without any warning as [@smacker said here](https://github.com/src-d/lookout/pull/484#discussion_r249486588), and that's also bad.

Reply:
I'd mention that it would be great if the error would be reported to us, but not a strong opinion; or maybe not needed to be done now/here.

Reply:
Let's keep that in mind for the future, at this stage I think it's enough that if something breaks at least you see why. How to report feedback can be done later when we have input from Product on the UI.